src/components/AppFrame.js

```diff
@@ -4,11 +4,17 @@ import { withStyles, MuiThemeProvider } from "@material-ui/core/styles";
 import Typography from "@material-ui/core/Typography";
 import AppBar from "@material-ui/core/AppBar";
 import Toolbar from "@material-ui/core/Toolbar";
+import Button from '@material-ui/core/Button';
 import IconButton from "@material-ui/core/IconButton";
+import SearchIcon from '@material-ui/icons/Search';
 import MenuIcon from "@material-ui/icons/Menu";
+import MenuItem from '@material-ui/core/MenuItem';
+import Menu from '@material-ui/core/Menu';
 import AppDrawer from "./AppDrawer";
 import { drawerTheme } from "../theme";
 
+const secondaryBarHeight = 30;
```

Comment:
This is the only I found to refactor this height instead of having it scattered in the style part of this component. Please let me know if there's another way to do this.